### PR TITLE
Fleet qa/test force update removal

### PR DESCRIPTION
**Pull request by mmartin24**
Deleting force update everywhere

### CODE REVIEW

File: tests/cypress/e2e/unit_tests/p1_fleet.spec.ts [hunk 1]
```diff
@@ -192,7 +192,6 @@ describe('Test Self-Healing of resource modification when correctDrift option us
       // Update exising GitRepo by enabling 'correctDrift'
       cy.addFleetGitRepo({ repoName, correctDrift: 'yes', editConfig: true });
       cy.clickButton('Save');
-      cy.open3dotsMenu(repoName, 'Force Update');
```
Comment (thehejik) [inline tests/cypress/e2e/unit_tests/p1_fleet.spec.ts:195]: maybe we should keep this ono in place as it may speedup the resources recovery

Comment (sbulage) [reply tests/cypress/e2e/unit_tests/p1_fleet.spec.ts:195]: yes, I am also proposing same. We can keep `Force Update` here for this test only.

**Comment by sbulage** [comment]
2.7 CI run: https://github.com/rancher/fleet-e2e/actions/runs/9560798251
2.9 CI run: https://github.com/rancher/fleet-e2e/actions/runs/9560809026

**Review by sbulage — APPROVED** [review]

**Comment by mmartin24** [comment]
Merging it. Let's monitor Fleet-77 for the upcoming days in case it needs to be reverted

**Pull request merged**